docs/source/models/intervention_models/neonatal/antibiotics_intervention.rst

```diff
@@ -114,19 +193,21 @@ Assumptions and Limitations
 ---------------------------
 
 - We assume that antibiotics availability captures actual use, and not simply the treatment being in the facility 
-- We assume that the delivery facility is the final facility in the care continum for deliveries that are transferred due to complications
+- We assume that the delivery facility is also the facility where a sick neonatal will seek care for sepsis
 - We assume that the relative risk of sepsis mortality with antibiotics in practice is a value that we can find in the literature
 - We have excluded the effect of antibiotics on pneumonia mortality, because this cause is currently lumped with 'other causes'
 
 Validation and Verification Criteria
 ------------------------------------
 
 - Population-level mortality rate should be the same as when this intervention is not included in the model
-- The ratio of sepsis deaths per birth among those without antibiotics access divided by those with antibiotics access should equal the relative risk parameter used in the model
+- The ratio of sepsis deaths per birth among those with antibiotics access divided by those without antibiotics access should equal the relative risk parameter used in the model
```

Comment:
I don't understand this change.

Reply:
I switched things above to be in terms of the relative risk of *not* having access to antibiotics, and this inverts the ratio to match that

Reply:
I got that but I still think you've got it backwards

Reply:
oh, you are right!
```suggestion
- The ratio of sepsis deaths per birth among those without antibiotics access divided by those with antibiotics access should equal the relative risk parameter used in the model
```